src/metadata/metadata_raw.c

```diff
@@ -645,6 +697,7 @@ static const struct raw_iface IRAW[metadata_raw_type_max] = {
 		.page			= _raw_ram_page,
 		.access			= _raw_ram_access,
 		.update			= _raw_ram_update,
+		.zero			= NULL,
```

Comment:
I believe it should be `raw_ram_zero()` instead of `NULL`. Otherwise we have nullptr dereference in attach for attomic mode. On top of that the configuration metadata in atomic mode is handled the same way as in normal, so without zeroing the superblock we'd have the same problem with potential metadata inconsistency.

Reply:
ok